tools/string_list.json

```
@@ -548,6 +548,7 @@
       "vertex_remove",
       "activate_add_ellipse_mode",
       "activate_add_line_mode",
+      "activate_add_poly_line_mode",
```

```suggestion
      "activate_add_polyline_mode",
```
Nitpick: it's `polyline` everywhere else.

---

ah yeah missed one spot because before I had it as `poly_line` everywhere. Thanks!